app/presenters/flat_reasons_for_rejection_presenter.rb

```diff
@@ -0,0 +1,72 @@
+      title_for_top_level_reason('offered_on_another_course_y_n') => y_n_to_boolean(reasons_for_rejection.offered_on_another_course_y_n),
+      title_for_top_level_reason('honesty_and_professionalism_y_n') => y_n_to_boolean(reasons_for_rejection.honesty_and_professionalism_y_n),
+      label_for_subreason('honesty_and_professionalism_y_n', 'information_false_or_inaccurate') => subreason_exists?(reasons_for_rejection.honesty_and_professionalism_concerns, 'information_false_or_inaccurate'),
+      label_for_subreason_detail('honesty_and_professionalism_y_n', 'information_false_or_inaccurate_details') => reasons_for_rejection.honesty_and_professionalism_concerns_information_false_or_inaccurate_details,
+      label_for_subreason('honesty_and_professionalism_y_n', 'plagiarism') => subreason_exists?(reasons_for_rejection.honesty_and_professionalism_concerns, 'plagiarism'),
+      label_for_subreason_detail('honesty_and_professionalism_y_n', 'plagiarism_details') => reasons_for_rejection.honesty_and_professionalism_concerns_plagiarism_details,
+      label_for_subreason('honesty_and_professionalism_y_n', 'references') => subreason_exists?(reasons_for_rejection.honesty_and_professionalism_concerns, 'references'),
+      label_for_subreason_detail('honesty_and_professionalism_y_n', 'references_details') => reasons_for_rejection.honesty_and_professionalism_concerns_references_details,
+      label_for_subreason_detail('honesty_and_professionalism_y_n', 'other_details') => reasons_for_rejection.honesty_and_professionalism_concerns_other_details,
+      title_for_top_level_reason('safeguarding_y_n') => y_n_to_boolean(reasons_for_rejection.safeguarding_y_n),
+      label_for_subreason('safeguarding_y_n', 'candidate_disclosed_information') => subreason_exists?(reasons_for_rejection.safeguarding_concerns, 'candidate_disclosed_information'),
+      label_for_subreason_detail('safeguarding_y_n', 'candidate_disclosed_information_details') => reasons_for_rejection.safeguarding_concerns_candidate_disclosed_information_details,
+      label_for_subreason('safeguarding_y_n', 'vetting_disclosed_information') => subreason_exists?(reasons_for_rejection.safeguarding_concerns, 'vetting_disclosed_information'),
+      label_for_subreason_detail('safeguarding_y_n', 'vetting_disclosed_information_details') => reasons_for_rejection.safeguarding_concerns_vetting_disclosed_information_details,
+      label_for_subreason_detail('safeguarding_y_n', 'other_details') => reasons_for_rejection.safeguarding_concerns_other_details,
+      title_for_top_level_reason('other_advice_or_feedback_y_n') => y_n_to_boolean(reasons_for_rejection.other_advice_or_feedback_y_n),
+      title_for_top_level_reason('interested_in_future_applications_y_n') => y_n_to_boolean(reasons_for_rejection.interested_in_future_applications_y_n),
+      'why are you rejecting this application details' => reasons_for_rejection.why_are_you_rejecting_this_application,
```

Comment:
This is a bit of a handful and would need work if we ever extend the model. I'd like to see a data declaration somewhere that allows this sort of thing to be generated, but that's for a follow-up PR I reckon.